src/py/flwr/client/app.py

```diff
@@ -224,7 +225,7 @@ def start_numpy_client(
         the same machine on port 8080, then `server_address` would be
         `"[::]:8080"`.
     client_fn : Optional[Callable[[str], NumPyClient]]
-        A callable that instantiates a NumPyClient. (default: None)
+        A callable that returns a client of type NumPyClient. (default: None)
```

Comment:
Should we just remove the support for `client_fn` from `start_numpy_client`? We added the capability after the Flower 1.5 stable release, which means it was never available to Flower stable users.

Reply:
yes, removed